src/main/java/net/pms/network/RequestV2.java

```diff
@@ -1107,7 +1107,13 @@ private StringBuilder browseHandler() {
 
 	private StringBuilder searchHandler() {
 		SearchRequest requestMessage = getPayload(SearchRequest.class);
-		return searchRequestHandler.createSearchResponse(requestMessage, mediaRenderer);
+
+		try {
+			return searchRequestHandler.createSearchResponse(requestMessage, mediaRenderer);
+		} catch (Exception e) {
+			LOGGER.trace("error transforming searchCriteria to SQL. Fallback to content browsing ...", e);
```

Comment:
To save the error `e` in the log is only for testing. It should be removed to not spam the log.